backend/migrate.sh

```
@@ -3,7 +3,7 @@
 echo "Waiting for postgres..."
 
 until nc -z "$POSTGRES_HOST" "$POSTGRES_PORT"; do
-  sleep 2
+  sleep 2
```

Shellcheck (0.11.0)

[error] This is a unicode space. Delete and retype it.

(SC1018)